tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -20,6 +20,10 @@ DEEPBUFFER_FW_DMA_MS=100"
 "cavs-sdw\;sof-mtl-rt711-4ch\;PLATFORM=mtl,NUM_DMICS=4,DMIC0_ID=2,DMIC1_ID=3,NUM_HDMIS=0,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-rt711-4ch.bin,DEEPBUFFER_FW_DMA_MS=100"
 
+"cavs-sdw\;sof-mtl-cs42l42-l0-max98363-l2-4ch\;PLATFORM=mtl,\
```

Comment:
how would the topology deal with the amps on link2, there's nothing that tells us to use link2 anywhere in the macros.

Reply:
Ack, think spk amp is on link2 by default now, maybe we should use macro to config it explicitly for future-proof: 
https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/platform/intel/sdw-amp-generic.conf

Reply:
> I think this topology is just confusing. I have no idea what the BE_ID refers to, if this is the index of the dailink or the physical link. @bardliao any comments?

@plbossart @yongzhi1 BE_ID refers to dailink index. SDW_SPK_STREAM refers to the physical link. Default string 'SDW1-Playback' is link 1. So, you probably need `SDW_SPK_STREAM=SDW2-Playback'`

Reply:
Ack, will update SDW_SPK_STREAM, but is this just a token referenced internally? fyi here is the config diagram from
from https://review.coreboot.org/c/coreboot/+/73280:
![sdw](https://user-images.githubusercontent.com/19287706/228912950-2c260697-5356-4a9a-9e9d-8f3b89cd91b7.gif)

Reply:
@yongzhi1 SDW_SPK_STREAM is not only referenced internally. It is also the name of Object.Dai.ALH and should match to the dai link name. For some reason, SDW_SPK_STREAM is referenced to copier's stream name, too. However, copier's stream name doesn't have to be the same as DAI name.